e2e/js_run_devserver/serve_test.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+# sedi makes `sed -i` work on both OSX & Linux
+# See https://stackoverflow.com/questions/2320564/i-need-my-sed-i-command-for-in-place-editing-to-work-with-both-gnu-sed-and-bsd
+_sedi () {
+  case $(uname) in
+    Darwin*) sedi=('-i' '') ;;
+    *) sedi=('-i') ;;
+  esac
+
+  sed "${sedi[@]}" "$@"
+}
+
+./node_modules/.bin/ibazel run "$1" >/dev/null 2>&1 &
```

Comment:
reminds me that your ibazel benchmark runner from like 5 years ago can be useful again to measure the new roundtrip

Reply:
good idea. I should revive that again and make some new ibazel benchmarks and a corresponding blog post to advertise this new rule